src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketPostBuildStatus.java

```diff
@@ -17,7 +17,7 @@
 
 class BitbucketPostBuildStatus extends GitSCMExtension {
 
-    private final JenkinsProvider jenkinsProvider;
+    private final transient JenkinsProvider jenkinsProvider;
```

Comment:
This does not work, this will make this throw an NPE when run (after a restart or on a remote slave).